udunits2/udunits2.spec

```diff
@@ -0,0 +1,243 @@
+Name: udunits2
+Version: 2.2.28
+Release: 7%{?dist}
+Summary: A library for manipulating units of physical quantities
+License: UCAR
+URL: http://www.unidata.ucar.edu/software/udunits/
+Source0: ftp://ftp.unidata.ucar.edu/pub/udunits/udunits-%{version}.tar.gz
+# default_udunits2_xml_path() is buggy and broken
+Patch0: udunits-2.2.28-fix-xml-path-logic.patch
+BuildRequires: make
+BuildRequires: gcc-c++, groff, byacc, expat-devel, CUnit-devel
+BuildRequires: chrpath
+BuildRequires: /usr/bin/makeinfo
```

Comment:
```suggestion
BuildRequires: texinfo
```

```
dnf provides makeinfo
Last metadata expiration check: 0:30:23 ago on Thu 08 Jun 2023 07:06:05 UTC.
texinfo-6.8-4.oe2203sp1.aarch64 : The GNU Documentation System
Repo        : @System
Matched from:
Filename    : /usr/bin/makeinfo
```

Reply:
I have some doubts at this point, and I don't quite understand the reason for replacing it with texinfo.thank you very much.

Reply:
From https://rpm-packaging-guide.github.io/ :
```
BuildRequires | A  comma- or whitespace-separated list of packages required for building  the program written in a compiled language. There can be multiple  entries of BuildRequires, each on its own line in the SPEC file.
```
Note the `list of packages`

Reply:
This seems to be a very practical learning material for me as a beginner, and I will study and practice it carefully, thanks to martin-g.